tests/scalability/README.md

```diff
@@ -94,3 +94,45 @@ From here we get the ddocs update sequences. Stage the branch which is associate
 Once the medic script completes we begin setting up for jmeter. The `start_jmeter_ec2.sh` script launches a new ec2 instance with the `user-data` script at `cht-core/tests/scalability/run_suite.sh`. 
 
 The script clones cht-core and navigates to the scalability dir. Then installs java, nodejs, jmeter, and its plugins. Executes the scalability suite which is defined in the `cht-core/tests/scalability/sync.jmx`, and finally it uploads the results to an s3 bucket. 
+
+
+
+## Distributed Workflow
+
+![Diagram of our current setup.](jmeter_distributed.png)
+
+Our current setup has 1 ec2 instance acting as the JMeter Orchestrator with 2 worker nodes. This can generate 1400 virtual users. Against our Application environment under test. 
+
+### Setup
+
+1. Launch an ec2 instance c5.2xlarge(Orchestrator)
+1. Launch 2 ec2 instances of c5.4xlarge(Worker)
+1. SSH into each of these machines. 
+1. Copy `tests/scalability/run_suite.sh` to the user directory. From this branch.
+1. On each of the three VMs, call  `chmod +x run_suite.sh&&./run_suite.sh` . This clones cht-core, installs JAVA, installs Node 14, runs npm install, installs JMeter, and its plugins.
+1. On just the orchestrator node, update the `cht-core/tests/scalability/config.json` to ensure the users and URL are correct.
+1. Split the `config.json` on the orchestrator
+    1. `cd cht-core/tests/scalability`
+    1. `npm install` 
```

Comment:
i had to do `sudo npm install -g grunt-cli` before `npm install` worked.

Reply:
hmm I don't know why you'd have to install grunt as this package.json doesn't have it as a dep.  I launched a fresh ubuntu container, installed node/npm,  cloned the repo, `cd cht-core/tests/scalability`, then `npm install`. Were you in a different directory or something? I'm not able to recreate this.

Reply:
ah - yup that was totally me - sorry! I was in `cht-core`.  Indeed, I just retested and being in `cht-core/tests/scalability` and `npm install` ran fine as did the `split_config.js` script!